src/Altinn.AccessManagement/Program.cs

```diff
@@ -199,15 +203,18 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
 
     GeneralSettings generalSettings = config.GetSection("GeneralSettings").Get<GeneralSettings>();
     services.Configure<GeneralSettings>(config.GetSection("GeneralSettings"));
-    services.Configure<PlatformSettings>(config.GetSection("PlatformSettings"));
+    services.Configure<PlatformSettings>(config.GetSection("GeneralSettings"));
```

Comment:
I belive this should be in section platformsettings if this is URL for profile component. Then config for platform should be moved in appsettings